### DISCUSSION TITLE
Icon for peak (heures pleines)/off-peak hours (heures creuses) for energy providers in France

### DISCUSSION THREAD

**Discussion by nanostra** [Mariusthvdb/custom-icons#61] [Icon suggestions/requests]
Hello, in France, we have electricity subscriptions that operate with peak and off-peak hours. To differentiate between consumption during these periods, I, like many French people, would greatly appreciate it if you could integrate these two icons: HC for off-peak hours (heures creuses) and HP for peak hours (heures pleines). For your information, EDF in the name of the icon is the historical electricity provider in France. Thank you in advance.

![edf-hc](https://github.com/user-attachments/assets/2a8aeea5-6ae1-4505-acfd-12f59be0515e)
![edf-hp](https://github.com/user-attachments/assets/175c29fb-3492-47cc-ba8f-19b4d3374320)

**Comment by Mariusthvdb** [comment]
thanks for reaching out

first of all: those are not svg icons? No way we could add them ike that, you would have to create those .svg icons

second: if you have those .svg icons, just use the custom_component (and not the js plugin) and add the icons to the folder, to make them available to the custom_component.

No more need to keep editing the JS script.

the CC is way easier and just requires the drop in of the icon

Ive added a section in the read me explaining this
